### ISSUE TITLE
Clarify why builder level is meaningful in threats

### ISSUE THREAD

**Issue by TomHennen** [slsa-framework/slsa#1215]
This is a great place to expand. Thanks for starting this.

I think the mitigation for this is essentially: 
* Have a way to know what "correct" looks like for this resourceUri. 
   * If you have access to such a thing, you can verify the artifact yourself by inspecting the data in the provenance attestation. 
   * If you do not, you must rely on a VSA with access to definition of "correct"
   * If you do not have either, you _cannot know_ whether the artifact is safe to use, even if it was SLSA Build L3

I think in the example below "build L3" is conflated with "built in the correct way" which is not necessarily the case.

_Originally posted by @zachariahcox in https://github.com/slsa-framework/slsa/pull/1191#discussion_r1806779663_

**Comment by TomHennen** [comment]
I think what we want to do is clarify in 'Mitigation' that the user setting a specific builder expectation protects them from vulnerabilities in other builders and a high level builder makes some vulnerabilities less likely.

We should probably update both G and F here.

**Comment by zachariahcox** [comment]
After talking it through, I think we can safely defer this discussion to 1.2. 

The summary is:

* The builder is the name of the build platform, as referenced in the "verifying build platforms" docs
* The builder _id_ is the "[transitive closure](https://github.com/slsa-framework/slsa/blob/c24955cc2227c80fb96991f77eb8e507e282ccee/docs/spec/draft/provenance.md?plain=1#L50)" of instructions that is fed to the builder.
* The verifier system should require a builder, an "id" https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/extends?view=azure-pipelines
(EG: a workflow file @ revision)

**Comment by TomHennen** [comment]
I don't think we need this to block the 1.2 RC release so I'm going to remove the 1.2 tag.
